### PR TITLE
1605/cannot edit terms color

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue

Addresses #1605

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Just makes the checkbox text blue so it's more visible

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

You don't need to fill out a whole application - locally just visit `http://localhost:3000/applications/review/terms`

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [x] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: ui-components/src/forms/FieldGroup.tsx [hunk 1]
```diff
@@ -59,7 +61,7 @@ const FieldGroup = ({
               defaultChecked={item.defaultChecked || false}
               ref={register(validation)}
             />
-            <label htmlFor={item.id} className="font-semibold">
+            <label htmlFor={item.id} className={`font-semibold ${fieldLabelClassName}`}>
```
Comment (seanmalbert) [inline ui-components/src/forms/FieldGroup.tsx:64]: @emilyjablonski , can you update the storybook story for this to show this new option?

Comment (emilyjablonski) [reply ui-components/src/forms/FieldGroup.tsx:64]: We don't have any stories for this component, I can add the file 👍

**Comment by emilyjablonski** [comment]
@seanmalbert story added 👍

**Review by seanmalbert — APPROVED** [review]
Thanks for the quick turnaround!

**Pull request merged**